RM_vs_DFN.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>Let's either uncomment it or delete it (happy either way)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/brosaplanella/formation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Uncomment it